service/history/task/transfer_active_task_executor.go

```diff
@@ -1790,17 +1793,27 @@ func (t *transferActiveTaskExecutor) processParentClosePolicy(
 				RunID:      childInfo.StartedRunID,
 				Policy:     childInfo.ParentClosePolicy,
 			})
+
+			if len(executions) == batchSize {
+				err := t.parentClosePolicyClient.SendParentClosePolicyRequest(ctx, parentclosepolicy.Request{
+					DomainName: domainName,
+					Executions: executions,
+				})
+				if err != nil {
+					return err
```

Comment:
retry attempt of the task will have to go thru each batch. is this operation idempotent?

Reply:
the operation signal a workflow and it is idempotent